### PR TITLE
DIG-4935 Remove 'never' option for recurring events in Drupal

### PR DESCRIPTION
**Pull request by subaha-cob**
DIG-4935 Remove 'never' option for recurring events in Drupal

### CODE REVIEW

File: docroot/themes/custom/bos_admin/css/bos_admin.css [hunk 1]
```diff
@@ -1197,15 +1197,7 @@ What: Helper for status_display preview - restricts icon sizing.
   margin-right: 0;
   margin-left: auto;
 }
-@media screen and (min-width: 840px)
-.g--6:last-child {
```
Comment (davidrkupton) [inline docroot/themes/custom/bos_admin/css/bos_admin.css:1201]: I am not sure why these classes are being removed.  They may not be needed for the datepicker, but other pages probably do use this style. I get removing and combining the extra duplicated & redundant `@media` directives, but I dont think we should be removing styles.

File: docroot/themes/custom/bos_admin/css/bos_admin.css [hunk 2]
```diff
@@ -1581,6 +1573,9 @@ Hide default contact info
   width: 48%;
   vertical-align: sub;
 }
+.js-form-item-field-event-date-recur-0-ends-mode.form-item-field-event-date-recur-0-ends-mode {
```
Comment (davidrkupton) [inline docroot/themes/custom/bos_admin/css/bos_admin.css:1576]: This is the only actual change we need to achieve the tickets requirement.

**Review by davidrkupton — CHANGES_REQUESTED** [review]
This looks OK. See my comments I am just concerned we are removing some css rules that should not be removed.

File: docroot/themes/custom/bos_admin/css/bos_admin.css (hunk 1 above)
Comment (subaha-cob) [reply docroot/themes/custom/bos_admin/css/bos_admin.css:1201]: Unless the error in the CSS was done on purpose, then all I did was fix the. missing tags in the css. The @media was mentioned several times, and they were all the same thing.

Comment (subaha-cob) [reply docroot/themes/custom/bos_admin/css/bos_admin.css:1201]: Screenshot 2024-10-02 at 12 00 57 PM

Comment (subaha-cob) [reply docroot/themes/custom/bos_admin/css/bos_admin.css:1201]: I replaced thee code above with this one:
Screenshot 2024-10-02 at 12 04 57 PM

**Review by subaha-cob — COMMENTED** [review]
Please see my comments.

**Pull request merged**